components/grid/row-drag-drop.md

```diff
@@ -68,6 +68,7 @@ This section contains the following examples:
 
 * [Drag and Drop a Row in the same Grid](#drag-and-drop-a-row-in-the-same-grid)
 * [Drag and Drop a Row between Grids](#drag-and-drop-a-row-between-grids)
+    * [Drag and Drop Rows in Grid Hierarchy](slug://grid-kb-drag-drop-rows-hierarchy)
```

Comment:
Note that you also list the examples on line 20. Is showing different lists here and there a deliberate decision?

Also, the rendering on this one is very ugly: 
![image](https://github.com/user-attachments/assets/b0606a57-47e2-42cd-8155-8b0141bf6338)

I recommend removing the indentation. Furthermore, it's quite odd to have all the items as bullet points and this one as a new page. The lack of consistency in the user experience is confusing. One possible solution is to mention it inline in the respective section. It could be a tip CTA panel instead of a link here.